src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -66,6 +66,37 @@ public function getDataFormSpecifications()
         return $tests;
     }
 
+    public function testLineBreaksBeforeDocumentSeparatorAndComment()
+    {
+        $yaml = <<<'EOF'
+
+ 
+# Two empty lines above (second - with one whitespace) and two below (first - with three whitespaces).
+   
+
```

Comment:
I can't say that this is the risk now, since YAML's correctness has an additional validation. But ok, I've did it.